cpp/CMakeLists.txt

```diff
@@ -292,6 +292,9 @@ include(cmake/thirdparty/get_cccl.cmake)
 # find rmm
 include(cmake/thirdparty/get_rmm.cmake)
 
+# find croaring
+include(cmake/thirdparty/get_croaring.cmake)
```

Comment:
I have removed `libroaring` from `cudf/conda/recipe.yaml` (`ignore_run_exports` list) since we are now only using it in tests and benchmarks. Do we need to revert that? @vyasr @bdice